packages/blade/src/components/Table/docs/APIStories/VirtualizedTableHoverActions.stories.tsx

```diff
@@ -0,0 +1,198 @@
+            </TableToolbarActions>
+          </TableToolbar>
+        }
+      >
+        {(tableData) => (
+          <VirtulizedWrapper
+            tableData={tableData}
+            rowHeight={(item, index) => {
+              console.log('item', item);
+              console.log('index', index);
+              return 48;
+            }}
+          >
+            <TableHeader>
+              <TableHeaderRow>
+                <TableHeaderCell>ID</TableHeaderCell>
+                <TableHeaderCell>Amount</TableHeaderCell>
+                <TableHeaderCell>Account</TableHeaderCell>
+                <TableHeaderCell>Date</TableHeaderCell>
+                <TableHeaderCell>Method</TableHeaderCell>
+                <TableHeaderCell>Status</TableHeaderCell>
+              </TableHeaderRow>
+            </TableHeader>
+            <TableBody<Item>>
+              {(tableItem, index) => (
+                <TableRow
+                  key={index}
+                  item={tableItem}
+                  isDisabled={index === 3}
+                  hoverActions={
+                    <>
+                      <IconButton
+                        accessibilityLabel="Copy"
+                        isHighlighted
+                        icon={CopyIcon}
+                        onClick={() => console.log('copy', tableItem)}
+                      />
+                      <IconButton
+                        accessibilityLabel="Delete"
+                        isHighlighted
+                        icon={TrashIcon}
+                        onClick={() => console.log('delete', tableItem)}
+                      />
+                    </>
+                  }
+                >
+                  <TableCell>
+                    <Code size="medium">{tableItem.paymentId}</Code>
+                  </TableCell>
+                  <TableCell>
+                    <Amount value={tableItem.amount} />
+                  </TableCell>
+                  <TableCell>{tableItem.account}</TableCell>
+                  <TableCell>
+                    {tableItem.date?.toLocaleDateString('en-IN', {
+                      year: 'numeric',
+                      month: '2-digit',
+                      day: '2-digit',
+                    })}
+                  </TableCell>
+                  <TableCell>{tableItem.method}</TableCell>
+                  <TableCell>
+                    <Badge
+                      size="medium"
+                      color={
+                        tableItem.status === 'Completed'
+                          ? 'positive'
+                          : tableItem.status === 'Pending'
+                          ? 'notice'
+                          : tableItem.status === 'Failed'
+                          ? 'negative'
+                          : 'default'
+                      }
+                    >
+                      {tableItem.status}
+                    </Badge>
+                  </TableCell>
+                </TableRow>
```

Comment:
Check with footer once

Reply:
checked VirtualizedTable does not support footer. we can only pass header and footer. also can't think of a case where we will be using VirtualizedTable with footer.